From 354117a8158999583bcfd44ac45cb658d6399087 Mon Sep 17 00:00:00 2001
From: evilwombat <evilwombat@server.fake>
Date: Mon, 9 Sep 2013 21:29:37 -0700
Subject: [PATCH 2/2] ARM: Ambarella: Enable ST7585 LCD platform device

Modify the SPI0.0 device declaration to use the st7585fb
framebuffer driver.

Signed-off-by: evilwombat <evilwombat@server.fake>
---
 arch/arm/mach-ambarella/init-generic.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-ambarella/init-generic.c b/arch/arm/mach-ambarella/init-generic.c
index afc1ce3..4437e08 100755
--- a/arch/arm/mach-ambarella/init-generic.c
+++ b/arch/arm/mach-ambarella/init-generic.c
@@ -164,7 +164,7 @@ static struct platform_device *ambarella_devices[] __initdata = {
 /* ==========================================================================*/
 static struct spi_board_info ambarella_spi_devices[] = {
 	{
-		.modalias	= "spidev",
+		.modalias	= "st7585fb",
 		.bus_num	= 0,
 		.chip_select	= 0,
 	},
-- 
1.7.9.5

